docroot/themes/humsci/su_humsci_theme/patterns/molecules/accordion/scss/accordion.scss

```diff
@@ -9,8 +9,8 @@ details {
     summary {
       border-bottom: 1px solid #ccc;
 
-      &:after {
-        content: "\f068"
+      &::after {
+        content: "\f068";
```

Comment:
Oh wow! I'm surprised the other linter didn't pick up on the missing semicolon. That seems kinda like it would be a big deal. 🤷

Reply:
right?!! 😂